modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -4228,6 +4190,13 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="ignoreErrorType" type="errorIdentifierType" use="optional">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            The full name of the error types or a comma separated list of full names, to ignore when processing an item. This is useful to use validations inside this component.
```

Comment:
Improve with something like: A comma separated list of error types for that should be handled by filtering the collection item being processed, such that it is excluded in the resulting collection, rather than propagating the error.